IPs/RSKIP252.md

```diff
@@ -0,0 +1,61 @@
+# Transaction Gas Price Cap
+
+|RSKIP          |252           |
+| :------------ |:-------------|
+|**Title**      |Transaction Gas Price Cap|
+|**Created**    |29-JUN-2021 |
+|**Author**     |SDL |
+|**Purpose**    |Sec, Fair |
+|**Layer**      |Core |
+|**Complexity** |1 |
+|**Status**     |Draft |
+|**discussions-to**     ||
+
+# **Abstract**
+
+This RSKIP proposes the limit of the transaction gas price as a factor of the minimum gas price established in the block header. The objective is twofold: remove incentives for [fee sniping](https://bitcoinops.org/en/topics/fee-sniping/) and [fee-whale transactions](https://medium.com/r/?url=https%3A%2F%2Fwww.cs.umd.edu%2F~jkatz%2Fpapers%2Fwhale-txs.pdf), and to prevent users from accidentally draining their wallets with unintended high transaction fees.
```

Comment:
Suggest using multiple instead of factor...    "This RSKIP proposes to limit a transaction's gas price to a multiple of the minimum.."